test/ui-test/src/screens/StatusChatScreen.py

```diff
@@ -192,13 +189,22 @@ def reply_to_message_at_index(self, index: int, message: str):
         message_object_to_reply_to = self.get_message_at_index(index)
         verify(not is_null(message_object_to_reply_to), "Message to reply to is loaded")
         move_mouse_over_object(message_object_to_reply_to)
-        click_obj_by_name(ChatComponents.REPLY_TO_MESSAGE_BUTTON.value)
+        foundreplyToButton = getChildrenWithObjectName(message_object_to_reply_to, "replyToMessageButton")[0]
```

Comment:
We use snake_case in python. Bonus points if you fix `getChildrenWithObjectName`
```suggestion
        found_reply_to_button = getChildrenWithObjectName(message_object_to_reply_to, "replyToMessageButton")[0]
```

Reply:
For bonus points always.